packages/create-plugin/templates/app/src/module.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Suspense, lazy } from 'react';
+import { AppPlugin, type AppRootProps } from '@grafana/data';
+import { LoadingPlaceholder } from '@grafana/ui';
+import type { AppConfigProps } from './components/AppConfig/AppConfig';
+
+const LazyApp = lazy(() => import('./components/App/App'));
+const LazyAppConfig = lazy(() => import('./components/AppConfig/AppConfig'));
+
+const App = (props: AppRootProps) => (
+  <Suspense fallback={<LoadingPlaceholder text="" />}>
+    <LazyApp {...props} />
+  </Suspense>
+);
+
+const AppConfig = (props: AppConfigProps) => (
+  <Suspense fallback={<LoadingPlaceholder text="" />}>
```

Comment:
Nice! Since these are pages, we could consider defining an app page [skeleton](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/utils/skeleton.tsx/#L39) rather than displaying the loading placeholder. But feel free to skip in this pr.